userguide/payment/includes/payment-api-overview.adoc

```diff
@@ -1,6 +1,16 @@
+=== API Overview
+
+Kill Bill's payment related API's can mainly be classified as follows:
+
+* *Payment Methods APIs* - Used for operations related to payment methods like adding/deleting a payment method, etc.
+* *Payment APIs* - Used for payment related operations. These are used by:
+** The Kill Bill core billing engine to charge for subscriptions
+** Kaui (the Kill Bill Admin UI) to handle refunds, chargebacks, adjustments, etc. for invoices and payments associated with such subscriptions. 
+** Directly by an e-commerce application to trigger payments for one-off charges (auth, capture, credit, refund, etc) or to create payment forms and redirect URLs in case of hosted payment pages.
```

Comment:
Maybe remove `(auth, capture, credit, refund, etc)` as `credit, refund` are not really 'charges' (or alternatively, we should change the word _charge_ -- maybe use _transaction_?).